forge-gui/res/cardsfolder/upcoming/struggle_for_project_purity.txt

```diff
@@ -0,0 +1,15 @@
+Name:Struggle for Project Purity
+ManaCost:3 U
+Types:Enchantment
+K:ETBReplacement:Other:SiegeChoice
+SVar:SiegeChoice:DB$ GenericChoice | Choices$ Brotherhood,Enclave | Defined$ You | SetChosenMode$ True | AILogic$ Brotherhood | ShowChoice$ ExceptSelf | SpellDescription$ As CARDNAME enters the battlefield, choose Brotherhood or Enclave.,,,• Brotherhood — At the beginning of your upkeep, each opponent draws a card. You draw a card for each card drawn this way.• Enclave — Whenever a player attacks you with one or more creatures, that player gets twice that many rad counters.
+SVar:Brotherhood:DB$ Animate | Defined$ Self | Triggers$ BrotherhoodTrig | Duration$ Permanent | SpellDescription$ Brotherhood
+SVar:BrotherhoodTrig:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Brotherhood — At the beginning of your upkeep, each opponent draws a card. You draw a card for each card drawn this way.
+SVar:TrigDraw:DB$ Draw | Defined$ Opponent | NumCards$ 1 | RememberDrawn$ True | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ Remembered$Amount | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:Enclave:DB$ Animate | Defined$ Self | Triggers$ EnclaveTrig | Duration$ Permanent | SpellDescription$ Enclave
+SVar:EnclaveTrig:Mode$ AttackersDeclaredOneTarget | AttackedTarget$ You | AttackingPlayer$ Player | Execute$ TrigRadiation | TriggerZones$ Battlefield | TriggerDescription$ Enclave — Whenever a player attacks you with one or more creatures, that player gets twice that many rad counters.
+SVar:TrigRadiation:DB$ Radiation | Defined$ TriggeredAttackingPlayer | Num$ X
+SVar:X:Count$TriggeredAttackers/Times.2
```

Comment:
that's not how you count triggered objects

Reply:
Any ideas ?

Reply:
`TriggerObjectsAttackers` when it's a Collection
should probably refactor that someday

Reply:
Thanks working 
![image](https://github.com/Card-Forge/forge/assets/67333662/98b64d2a-2be7-4785-a133-314a54240f4e)